ftd/built-in-types.ftd

```diff
@@ -823,7 +823,7 @@ background image based on the specified x and y values.
 
 
 
---ds.h2: `ftd.fetch-priority`
+-- ds.h2: `ftd.fetch-priority`
 
 The `ftd.fetch-priority` property of the HTMLImageElement interface represents a hint given to the browser on
 how it should prioritize the fetch of the image relative to other images.
```

Comment:
You lines are still too long. Look at the PR on Github, long lines reduces readability in source code, things get wrapped, and does not look good.